vocabularies/licence.ttl

```diff
@@ -0,0 +1,52 @@
+@prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos:  <http://www.w3.org/2004/02/skos/core#> .
+@prefix reg:   <http://purl.org/linked-data/registry#> .
+@prefix dct:   <http://purl.org/dc/terms/> .
+@prefix ldp:   <http://www.w3.org/ns/ldp#> .
+@prefix owl:   <http://www.w3.org/2002/07/owl#> .
+@prefix rr-org: <http://registry.it.csiro.au/structure/org/> .
+
+
+<http://registry.it.csiro.au/licence/cc-by-nd-4.0>
+        a                     reg:Delegated , reg:NamespaceForward , skos:Concept ;
+        skos:inScheme <http://registry.it.csiro.au/licence> ;
+        skos:topConceptOf <http://registry.it.csiro.au/licence> ;
+        rdfs:label            "Creative Commons Attribution-NoDerivatives 4.0 International"@en ;
+        skos:prefLabel        "Creative Commons Attribution-NoDerivatives 4.0 International"@en ;
+        dct:description       "You are free to: Share - copy and redistribute the material in any medium or format for any purpose, even commercially. The licensor cannot revoke these freedoms as long as you follow the licence terms. Under the following terms: Attribution - You must give appropriate credit, provide a link to the licence, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use. NoDerivatives - If you remix, transform, or build upon the material, you may not distribute the modified material. No additional restrictions - You may not apply legal terms or technological measures that legally restrict others from doing anything the licence permits Notices - You do not have to comply with the licence for elements of the material in the public domain or where your use is permitted by an applicable exception or limitation. No warranties are given. The licence may not give you all of the permissions necessary for your intended use. For example, other rights such as publicity, privacy, or moral rights may limit how you use the material."@en ;
```

Comment:
```suggestion
        dct:description       "You are free to: Share - copy and redistribute the material in any medium or format for any purpose, even commercially. The licensor cannot revoke these freedoms as long as you follow the licence terms. Under the following terms: Attribution - You must give appropriate credit, provide a link to the licence, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use. NoDerivatives - If you remix, transform, or build upon the material, you may not distribute the modified material. No additional restrictions - You may not apply legal terms or technological measures that legally restrict others from doing anything the licence permits Notices - You do not have to comply with the licence for elements of the material in the public domain or where your use is permitted by an applicable exception or limitation. No warranties are given. The licence may not give you all of the permissions necessary for your intended use. For example, other rights such as publicity, privacy, or moral rights may limit how you use the material."@en ;
        skos:definition       "You are free to: Share - copy and redistribute the material in any medium or format for any purpose, even commercially. The licensor cannot revoke these freedoms as long as you follow the licence terms. Under the following terms: Attribution - You must give appropriate credit, provide a link to the licence, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use. NoDerivatives - If you remix, transform, or build upon the material, you may not distribute the modified material. No additional restrictions - You may not apply legal terms or technological measures that legally restrict others from doing anything the licence permits Notices - You do not have to comply with the licence for elements of the material in the public domain or where your use is permitted by an applicable exception or limitation. No warranties are given. The licence may not give you all of the permissions necessary for your intended use. For example, other rights such as publicity, privacy, or moral rights may limit how you use the material."@en ;
```

I have to admit I'm not sure if our VocPrez instance will pick up on dct:description, or if it will only display a skos:definition. You may need both for each concept if that is the case. Do you know the answer, @DavidCrosswellGSQ?

Reply:
@KellyVance could we stick with calling it 'Licence' or something similar. We will likely have additional licence types to add in the future that won't necessarily be Creative Commons.

Reply:
That might be a good approach. Can always create a _Creative Commons_ collection to pre-empt any potential expansion.

Reply:
I can't see how to commit part of the suggestion, will commit entirely and then revert the label back to _licence_

Reply:
Sorry, I can't comment on any of this. But, I assume that any licence/s that is/are set-up would need to be run past the departmental legal people. Back in 2013, when I published a paper in a Taylor & Francis journal who would normally hold the copyright, I overrode this to obtain Crown copyright, which was not unusual. But, I was blown away by the detail that the legal people drew up.

Reply:
We use CC licences for just about everything. If we do have other specialised licences they are done by the business groups and with legal input.

Reply:
All good. I was just noting Vance's comment that we will likely have additional licence types in the future that won't necessarily be Creative Commons.